ouroboros-consensus-cardano/test/Test/ThreadNet/Cardano.hs

```diff
@@ -138,9 +138,10 @@ instance Arbitrary TestSetup where
                 -- Shelley epoch, since stake pools can only be created and
                 -- delegated to via Shelley transactions.
                 `suchThat` ((/= 0) . Shelley.decentralizationParamToRational)
-    setupK <- SecurityParam <$> choose (4, 6)
-                -- If k < 4, common prefix violations become too likely in
-                -- Praos mode.
+    setupK <- SecurityParam <$> choose (8, 10)
+                -- If k < 8, common prefix violations become too likely in
+                -- Praos mode for thin overlay schedules (ie low d), even for
+                -- f=0.2.
```

Comment:
Could you measure how much longer the (non-nightly) tests now take?

Reply:
The ThreadNet tests change as follows:

```
BEFORE   107.57s   96.51s    103.83s   99.63s

AFTER    223.75s   219.25s   229.55s   252.89s
```

So it's about 130% slower (which is similar to the nightly changes I made).

Option A) I'll set the non-nightly to use only 40 instead of 100.

Option B) Alter the test setup to reach the 3rd epoch less often in the non-nightly run.

I'm favoring A, for simplicity.

Reply:
Yeah, A is fine for now.